socorro/scripts/fetch_crashids.py

```diff
@@ -141,24 +144,27 @@ def main(argv=None):
         description=DESCRIPTION.strip(),
     )
     parser.add_argument(
-        '--date', default='yesterday',
-        help='Date to pull crash ids from (YYYY-MM-DD)'
+        '--date', default='',
+        help=(
+            'date to pull crash ids from as YYYY-MM-DD, "yesterday", "today", or "now"; '
+            'defaults to "yesterday"'
+        )
```

Comment:
The previous help text was missing stuff. Plus the default doesn't show up when you do `--help`. I changed the default to empty string here because it makes figuring out what's going on easier later on.